Library/Homebrew/style.rb

```diff
@@ -178,6 +180,17 @@ def run_shellcheck(files, output_type)
         *files,
       ]
 
+      if fix
+        # patch options:
+        #   --get=0       : suppress environment variable `PATCH_GET`, ignore RCS, ClearCase, Perforce, and SCCS
+        #   --force       : we know what we are doing, force apply patches
+        #   --directory=/ : change to root directory, since we use absolute file paths
+        #   --strip=0     : do not strip path prefixes, since we are at root directory
+        patch_command = %w[patch --get=0 --force --directory=/ --strip=0]
+        patches = system_command(shellcheck, args: ["--format=diff", *args]).stdout
+        Utils.popen_write(*patch_command) { |p| p.write(patches) }
```

Comment:
```suggestion
        Utils.safe_popen_write(*patch_command) { |p| p.write(patches) }
```

Probably as we don't want to ignore failures, right?